I'm sorry, but an uncaught exception occurred.

While parsing C:\Users\Алхасов А А\Desktop\Визуал\Stuff\game/script.rpy.
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xc2 in position 2: invalid continuation byte

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "C:\Program Files (x86)\renpy-8-3-1-ru-portable-x64\renpy\bootstrap.py", line 359, in bootstrap
    renpy.main.main()
  File "C:\Program Files (x86)\renpy-8-3-1-ru-portable-x64\renpy\main.py", line 470, in main
    renpy.game.script.load_script() # sets renpy.game.script.
  File "C:\Program Files (x86)\renpy-8-3-1-ru-portable-x64\renpy\script.py", line 359, in load_script
    self.load_appropriate_file(".rpyc", [ "_ren.py", ".rpy" ], dir, fn, initcode)
  File "C:\Program Files (x86)\renpy-8-3-1-ru-portable-x64\renpy\script.py", line 929, in load_appropriate_file
    data, stmts = self.load_file(dir, fn + source)
  File "C:\Program Files (x86)\renpy-8-3-1-ru-portable-x64\renpy\script.py", line 721, in load_file
    stmts = renpy.parser.parse(fullfn)
  File "C:\Program Files (x86)\renpy-8-3-1-ru-portable-x64\renpy\parser.py", line 1676, in parse
    lines = list_logical_lines(fn, filedata, linenumber)
  File "C:\Program Files (x86)\renpy-8-3-1-ru-portable-x64\renpy\lexer.py", line 298, in list_logical_lines
    data = f.read().decode("utf-8", "python_strict")
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xc2 in position 2: invalid continuation byte

Windows-10-10.0.19045 AMD64
Ren'Py 8.3.1.24090601
Stuff 1.0
Fri Jan 17 13:30:12 2025
